jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/definition/details/ProcessDefinitionDetailsTabViewImpl.java

```diff
@@ -0,0 +1,176 @@
+import elemental2.dom.HTMLLabelElement;
+import elemental2.dom.HTMLParagraphElement;
+import org.jboss.errai.ui.shared.api.annotations.DataField;
+import org.jboss.errai.ui.shared.api.annotations.Templated;
+import org.jbpm.workbench.pr.client.resources.i18n.Constants;
+import org.uberfire.workbench.events.NotificationEvent;
+
+@Dependent
+@Templated
+public class ProcessDefinitionDetailsTabViewImpl extends Composite implements
+                                                                   ProcessDefinitionDetailsTabPresenter.AdvancedProcessDefDetailsView {
+
+    @Inject
+    @DataField
+    public HTMLParagraphElement nroOfHumanTasksText;
+
+    @Inject
+    @DataField
+    public HTMLParagraphElement humanTasksListBox;
+
+    @Inject
+    @DataField
+    public HTMLParagraphElement usersGroupsListBox;
+
+    @Inject
+    @DataField
+    public HTMLParagraphElement processDataListBox;
+
+    @Inject
+    @DataField
+    public HTMLParagraphElement processServicesListBox;
+
+    @Inject
+    @DataField
+    public HTMLParagraphElement subprocessListBox;
+
+    @Inject
+    @DataField
+    public HTMLLabelElement nroOfHumanTasksLabel;
+
+    @Inject
+    @DataField
+    public HTMLLabelElement humanTasksListLabel;
+
+    @Inject
+    @DataField
+    public HTMLLabelElement usersGroupsListLabel;
+
+    @Inject
+    @DataField
+    public HTMLLabelElement subprocessListLabel;
+
+    @Inject
+    @DataField
+    public HTMLLabelElement processDataListLabel;
+
+    @Inject
+    @DataField
+    public HTMLLabelElement processServicesListLabel;
+
+    @Inject
+    @DataField
+    protected HTMLParagraphElement processIdText;
+
+    @Inject
+    @DataField
+    protected HTMLParagraphElement processNameText;
+
+    @Inject
+    @DataField
+    protected HTMLParagraphElement deploymentIdText;
+
+    @Inject
+    @DataField
+    protected HTMLLabelElement processNameLabel;
+
+    @Inject
+    @DataField
+    protected HTMLLabelElement processIdLabel;
+
+    @Inject
+    @DataField
+    protected HTMLLabelElement deploymentIdLabel;
+
+    @Inject
+    private Event<NotificationEvent> notification;
+
+    private Constants constants = Constants.INSTANCE;
+
+    @PostConstruct
+    public void init() {
+        processIdLabel.textContent = constants.Process_Definition_Id();
+        processNameLabel.textContent = constants.Process_Definition_Name();
+        nroOfHumanTasksLabel.textContent = constants.Human_Tasks_Count();
+        deploymentIdLabel.textContent = constants.Deployment_Name();
+        humanTasksListLabel.textContent = constants.Human_Tasks();
+        usersGroupsListLabel.textContent = constants.User_And_Groups();
+        subprocessListLabel.textContent = constants.SubProcesses();
+        processDataListLabel.textContent = constants.Process_Variables();
+        processServicesListLabel.textContent = constants.Services();
+    }
+
+    @Override
+    public void setNroOfHumanTasksText(final String text) {
```

Comment:
A bit weird cutting :)
I would go with the whole word. It deosn't add much in length.

Reply:
Renamed to numberOf